model_drafts/tiles/tile.js

```diff
@@ -0,0 +1,25 @@
+var Sequelize = require('sequelize');
+
+function Tile(db){
+  var Tile = db.define('Tile', {
+    edgeN: Sequelize.UUID,
+    edgeE: Sequelize.UUID,
+    edgeS: Sequelize.UUID,
+    edgeW: Sequelize.UUID,
+    // edges: Sequelize.ARRAY(Sequelize.UUID), array of 0-3, alternative to separate NESW
+    floor: Sequelize.UUID, //null if it is a pit? what if it is just plain cement?
```

Comment:
With pits, there are different types of pits. A pit with a wall (which could be on any side) or a black cell with no walls. I'm not sure how we want to address that. We can discuss.